### PR TITLE
Integrate validation script to snakemake

### PR DESCRIPTION
**Pull request by yerbol-akhmetov**
Good day. Here, I propose to integrate `data_validation.py` script into `snakemake` workflow.

### CODE REVIEW

**Comment by yerbol-akhmetov** [comment]
One thing also to point, currently clusters wildcards does essentially nothing. But I have added it for the future. After I figure out advanced snakemake workflow, we will adjust the snakemake inputs.

File: plots/data_validation.py [hunk 1]
```diff
@@ -3,14 +3,18 @@
 # SPDX-License-Identifier: AGPL-3.0-or-later
 
 import os
+import sys
+sys.path.append(os.path.abspath(os.path.join(__file__ ,"../../")))
 import pandas as pd
 import numpy as np
```
Comment (GbotemiB) [inline plots/data_validation.py:9]: HI @yerbol-akhmetov, just a minor comment to remove numpy as it is not currently being used.

**Comment by GbotemiB** [comment]
@yerbol-akhmetov, Great Job. This looks good to me.

Just another minor comment. Since we are still setting up the repo, can we also have a test folder, even if it is empty.

**Comment by yerbol-akhmetov** [comment]
> @yerbol-akhmetov, Great Job. This looks good to me.
> 
> Just another minor comment. Since we are still setting up the repo, can we also have a test folder, even if it is empty.

Yes, we can have it. Should I commit it to this PR?

**Comment by yerbol-akhmetov** [comment]
@GbotemiB, @danielelerede-oet. Here I made few changes. In particular renamed `_helpers.py` to `_helper.py`, so it will not be confused with `pypsa-earth/scripts/_helpers.py`. I was just testing more complex snakemake approach with importing rules from pypsa-earth repo, so when importing and adding path to `sys.path`, it was sometimes confused. In addition, I have renamed the `config.plot.yaml` to `config.main.yaml` as it will be config not only for plots, but for parameters storage for our custom rules.

**Review by danielelerede-oet — CHANGES_REQUESTED** [review]
Thank you @yerbol-akhmetov , once the comment by @GbotemiB is addressed this is good to go :)

**Review by yerbol-akhmetov — COMMENTED** [review]
Thanks for the comment!

**Review by yerbol-akhmetov — COMMENTED** [review]
Can you please merge it @GbotemiB, if everything is fine?

**Comment by GbotemiB** [comment]
@yerbol-akhmetov, Merging right away 🚀
